### DISCUSSION TITLE
Virtual temperature instead of actual temperature in HA

### DISCUSSION THREAD

**Discussion by Bryla2507** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#550] [Q&A]
Hi. Yesterday I've calibrated my virtual temperature and it's working great, it's pretty accurate. Having that in mind, I'd like to use it instead of actual temperature in climate entity in HA, so when I disable the pump I can still read real water temperature. Is that possible in any way?

**Comment by visualapproach** [comment]
Hi, I haven't tried it but I guess it would just need the virtual temp sensor instead of the ordinary. For now you either need to change the code in HA.txt but it will be overwritten next update you make so maybe best option is to make a new climate entity in HA. I don't have time to do it right now but there are other people here that may want to help you with an example.

**Reply by visualapproach** [reply]
I checked it out but it turns out that the HVAC climate control is only displaying whole numbers even if I set the precision to 0.1
I think it's a limitation in that card. And even if it would've shown decimals, there is no way to set it in the ESP yet.
